### PR TITLE
#4911: updated changelog/readme

### PR DESCRIPTION
**Pull request by sreuland**
updated the readme and changelog notes for new reingestion for remote storage options.

### CODE REVIEW

File: services/horizon/internal/ingest/README.md [hunk 1]
```diff
@@ -140,8 +140,45 @@ This pauses the state machine for 10 seconds then tries again, in hopes that a n
...
+By default this sub-command will attempt to use captive core configuration in the form of stellar core binary(`--stellar-core-binary-path`) and stellar core config(`--captive-core-config-path`) to obtain ledger tx meta from a stellar network to be ingested. 
+
+The `db reingest range` sub-command can optionally be configured to consume pre-computed ledger tx meta files from a Google Cloud Storage(GCS) location instead of running captive core on host machine. 
+Pre-requirements: 
+  - Have a GCS account.
+  - Run the [ledgerexporter] to publish ledger tx meta files to your GCS bucket location.
+Run the `db reingest` sub-command, configured to import tx meta from your GCS bucket:
+  ```$ DATABASE_URL=<your db host dsn> \
+     NETWORK=testnet \
+     stellar-horizon db reingest range \
```
Comment (urvisavla) [inline services/horizon/internal/ingest/README.md:155]: Should we add `<from_ledger> <to_ledger>` in the command?

**Review by urvisavla — APPROVED** [review]
Looks great 👍

File: services/horizon/internal/ingest/README.md (hunk 1 above)
Comment (sreuland) [reply services/horizon/internal/ingest/README.md:155]: yes, done, thanks.

File: services/horizon/internal/ingest/README.md [hunk 2]
```diff
@@ -140,8 +140,45 @@ This pauses the state machine for 10 seconds then tries again, in hopes that a n
...
+		   --parallel-workers, 2 \
+		   --ledgerbackend, "datastore" \
+		   --datastore-config, "config.storagebackend.toml",
+  ```  
+Notice, even though we no longer need to provide stellar-core related config for binary or config file, we do still need to provide network related config, using convenience parameter `NETWORK=testnet|pubnet` or directly with `NETWORK_PASSPHRASE` and `HISTORY_ARCHIVE_URLS`
+
+The `--datastore-config` must point to a new toml config file that will provide the necessary parameters for ingestion to work with remote GCS storage.
+
+example config toml:
```
Comment (urvisavla) [inline services/horizon/internal/ingest/README.md:165]: Missing buffered_storage_backend_config
```
[buffered_storage_backend_config]
buffer_size = 5 # The size of the buffer
num_workers = 5      # Number of workers
retry_limit = 3      # Number of retries allowed
retry_wait = "30s"    # Duration to wait before retrying in seconds
```

**Pull request merged**